### DISCUSSION TITLE
Call blocks

### DISCUSSION THREAD

**Discussion by EpicEclipseGames** [TurboWarp/extensions#1415] [Ideas]
Can someone write an extension, which able turbowarp set calls enable. Like a walkey talkey program, just better.

**Comment by snowboyz0825** [comment]
Like an extension that allows you to make calls or something else? If you want that you _might_ be able to get something working with cloud variables, but good luck with that.

**Reply by EpicEclipseGames** [reply]
Yes! You're right!
